internal/clientsholder/clientsholder.go

```diff
@@ -157,12 +157,16 @@ func GetClientsHolder(filenames ...string) *ClientsHolder {
 		return &clientsHolder
 	}
 	if len(filenames) == 0 {
-		log.Error("Please provide a valid Kubeconfig. Either set the KUBECONFIG environment variable or alternatively copy a kube config to $HOME/.kube/config")
+		errMsg := "Please provide a valid Kubeconfig. Either set the KUBECONFIG environment variable or alternatively copy a kube config to $HOME/.kube/config"
+		log.Error(errMsg)
+		fmt.Fprintf(os.Stderr, "ERROR: %s\n", errMsg)
```

Comment:
Because it's an actual error what's been printed. It's an error to run the CNF Certification Suite without a valid Kubeconfig. Also, we don't use the old `log` package anymore, we have our own log package that implements an `slog` Handler.

Reply:
@jmontesi OK, good.